### ISSUE TITLE
Should warn about using `@extend` with a non-placeholder selectors

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#24]
Using `@extend` with standard selectors (non-placeholders) is considered dangerous. Placeholders should always be used.

Don't do this:

```
@extend .my-class;
```

Do this:

```
@extend %my-placeholder;
```

More information:

  http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#placeholders
  http://8gramgorilla.com/mastering-sass-extends-and-placeholders/
